tensorbay/opendataset/HKD/loader.py

```diff
@@ -96,6 +95,8 @@ def HKD(path: str) -> Dataset:
                 image_path = os.path.join(
                     segment_path, "CropImages", image_name_template.format(csv_line.pop(0))
                 )
+                if "NaN" in csv_line:
```

Comment:
Should the data with "NaN" label value be added to the segment?
And what about the data with more than one set of keypoints2d?

Reply:
solution is :
the data with NaN will not be added to the segment except this data has another set of annotations without NaN value
and we will add all sets of keypoints2d to the data as long as the value of keypoints2d is not NaN.